src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +62,25 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// content generated by function
+export function DynamicRowGeneration = (rowElementMaxCount, labelArray) => {
```

Comment:
You don't need to have `export function` here. A simple `const dynamicRowGeneration` will work here.

You use the `export` keyword if you want the function to be used in another file (so that other file can 'import' the function into its own scope).

Another note: Functions follow camel casing while components (the functions with props) are sort of title cased (each word is capitalized, but there's no spaces between words).